website/src/pages/en/subgraphs/_meta.js

```diff
@@ -6,6 +6,6 @@ export default {
   querying: titles.querying,
   developing: titles.developing,
   billing: '',
-  cookbook: titles.cookbook,
+  guides: 'How-to Guides',
```

Comment:
Folder titles should really live in the `_meta-titles.json` files to be translatable. 🙏

Reply:
Ah, looks like you already added it there. So:
```suggestion
  guides: titles.guides,
```
You may need to add it to all the languages, though, otherwise this will be `undefined` for some of them and I think that will break the build (maybe that's why you hardcoded it as a string here?)

Reply:
Oh, you added it to the wrong `_meta-titles.json`. 😅 It should be in the one in the `subgraphs` folder, and there should already be a `cookbook: 'Cookbook'` entry there for you to change.